node/src/main/scala/coop/rchain/node/Main.scala

```diff
@@ -1,30 +1,32 @@
 package coop.rchain.node
 
+import java.nio.file.{Path, Paths}
 import java.security.Security
 
 import coop.rchain.shared.StringOps._
 import cats.implicits._
-import scala.tools.jline.console._, completer.StringsCompleter
-import scala.collection.JavaConverters._
 
+import scala.tools.jline.console._
+import completer.StringsCompleter
+import scala.collection.JavaConverters._
 import coop.rchain.comm._
 import coop.rchain.casper.util.comm.{DeployRuntime, DeployService, GrpcDeployService}
 import coop.rchain.node.effects.{ConsoleIO, GrpcReplClient, ReplClient}
 import coop.rchain.catscontrib.TaskContrib._
 import coop.rchain.catscontrib._
 import coop.rchain.crypto.codec.Base16
-
 import monix.eval.Task
 import monix.execution.Scheduler
 import monix.execution.schedulers.SchedulerService
-import scala.util.{Failure, Success, Try}
 
+import scala.util.{Failure, Success, Try}
 import org.bouncycastle.jce.provider.BouncyCastleProvider
 
 object Main {
 
   def main(args: Array[String]): Unit = {
     val conf = Conf(args)
+    println(s"Starting with profile ${conf.profile.toOption.map(_.name)}")
```

Comment:
Shouldn't this be logged at info level?

Reply:
from what I've seen the approach is to `println` the messages targeted at the user
vide:
https://github.com/rchain/rchain/blob/faceaf7c8c5f2407bd6ccf838220ba5be0b9bbaa/node/src/main/scala/coop/rchain/node/node.scala#L72